components/support/sql/src/interrupt.rs

```diff
@@ -60,6 +60,11 @@ impl SqlInterruptScope {
         let start_value = ptr.load(Ordering::SeqCst);
         Self { start_value, ptr }
     }
+    /// Add this as an inherent method to reduce the amount of things users have to bring in.
+    #[inline]
+    pub fn err_if_interrupted(&self) -> Result<(), interrupt::Interrupted> {
```

Comment:
This means a bunch of files no longer need `use interrupt::Interruptee` which seems like a win to me.

Reply:
+1